src/openforms/logging/logevent.py

```diff
@@ -170,6 +170,16 @@ def prefill_retrieve_success(submission: "Submission", plugin, prefill_fields):
     )
 
 
+def prefill_retrieve_empty(submission: "Submission", plugin, prefill_fields):
+    _create_log(
+        submission,
+        "prefill_retrieve_empty",
```

Comment:
the template for this logevent is missing in the `src/openforms/logging/templates/logging/events/` directory